disable-checkpoint.py

```diff
@@ -21,13 +21,16 @@
 
 update_failures = []
 for probe in probe_list:
-    print("Removing checkpoint " + str(checkpoint["Id"])
-          + " from probe " + probe["Name"])
-    try:
-        probes.removeCheckpoint(checkpoint["Id"], probe, user, secret)
-    except exceptions.HTTPError as err:
-        print("Failed to remove checkpoint from probe " + probe["Name"] + " - " + str(err))
-        update_failures.append(probe)
+    if "Regions" in probe["SelectedCheckpoints"]:
+        print("Probe " + probe["Name"] + " is using region(s) instead of individual checkpoints - skipping")
```

Comment:
If we have monitors set up like this, won't that limit our ability to turn off individual checkpoints when they have individual checkpoint issues?

Reply:
It will, yes.  There are a few in under our account that are set up in this manner, though.  Those should probably be converted to match the others, as I don't know why they were set up that way.  However, since I went with the assumption that others outside of our org might use these scripts as well, I decided to skip them if they explicitly set up regional-level sets of checkpoints.  It might be worth calling this out after the script runs so it's pointed out to the user that "monitors x, y, and z weren't updated because you're using regions?"